File: assets/js/metall.js
var nextArrow = '<div class="control-next-arrow">▶▶</div>';
var prevArrow = '<div class="control-prev-arrow">◀◀</div>';
var maxSlidePosition = 7;

var control = {
    0: {
        id: "first-promo",
        title: "Фиброцементный сайдинг: предложение",
        prev: "",
        next: "Виды покрытий" + nextArrow,
    },
    1: {
        id: "summary",
        title: "Фиброцементный сайдинг: обзор",
        prev: prevArrow + "Предложение",
        next: "Палитра цветов" + nextArrow,
    },
    2: {
        id: "palitra",
        title: "Металлосайдинг: палитра цветов",
        prev: prevArrow + "Обзор",
        next: "Качества" + nextArrow,
    },
    3: {
        id: "kachestva",
        title: "Металлосайдинг: качества",
        prev: prevArrow + "Палитра цветов",
        next: "Технология монтажа" + nextArrow,
    },
    4: {
        id: "schema",
        title: "Металлосайдинг: монтаж",
        prev: prevArrow + "Качества",
        next: "Устройство откосов" + nextArrow,
    },
    5: {
        id: "otkos",
        title: "Металлосайдинг: откосы",
        prev: prevArrow + "Технология монтажа",
        next: "Портфолио" + nextArrow,
    },
    6: {
        id: "portfolio",
        title: "Металлосайдинг: портфолио",
        prev: prevArrow + "Устройство откосов",
        next: "Видео обзор" + nextArrow,
    },
    7: {
        id: "video",
        title: "Металлосайдинг: видео обзор",
        prev: prevArrow + "Портфолио",
        next: "Предложение" + nextArrow,
    },
}

var maxPortfolioPosition = 7;

var portfolio = {
    0: 'url(/assets/img/metal/portfolio/0.jpg)',
    1: 'url(/assets/img/metal/portfolio/1.jpg)',
    2: 'url(/assets/img/metal/portfolio/2.jpg)',
    3: 'url(/assets/img/metal/portfolio/3.jpg)',
    4: 'url(/assets/img/metal/portfolio/4.jpg)',
    5: 'url(/assets/img/metal/portfolio/5.jpg)',
    6: 'url(/assets/img/metal/portfolio/6.jpg)',
    7: 'url(/assets/img/metal/portfolio/7.jpg)',
}

File: views/metall.go
package views

import (
	"html/template"
	"net/http"

	mobiledetect "github.com/Shaked/gomobiledetect"
	"github.com/meehalkoff/loggi"
	"gitlab.com/mvalkov/katzman/lib"
)

func MetallHandler(w http.ResponseWriter, r *http.Request) {

	tmpl := template.New("metall")
	detect := mobiledetect.NewMobileDetect(r, nil)
	if detect.IsMobile() {
		tmpl.ParseFiles(
			"mobile/templates/metal.html",
			"mobile/templates/header.html",
			"mobile/templates/footer.html",
			"mobile/templates/top-panel.html",
			"mobile/templates/calc.html",
			"mobile/templates/call.html",
			"mobile/templates/about.html",
			"mobile/templates/firstPromoBlock.html",
			"mobile/templates/creditPromoBlock.html",
			"mobile/templates/lastPromoBlock.html",
			"mobile/templates/how-we-work.html",
		)
	} else {
		tmpl.ParseFiles(
			"templates/header.html",
			"templates/footer.html",
			"templates/top-panel/top-panel.html",
			"templates/top-panel/about.html",
			"templates/top-panel/calc.html",
			"templates/top-panel/call.html",
			"templates/top-panel/main-menu.html",
			"templates/control-panel.html",
			"templates/firstPromoBlock.html",
			"templates/metall/index.html",
			"templates/metall/summary.html",
			"templates/metall/palitra.html",
			"templates/metall/kachestva.html",
			"templates/metall/schema.html",
			"templates/metall/otkos.html",
			"templates/howWeWork.html",
			"templates/metall/portfolio.html",
			"templates/shared/video.html",
			"templates/metall/promo.html",
			"templates/lastPromoBlock.html",
			"templates/creditPromoBlock.html",
		)
	}
	data := lib.FirstPromoBlock{
		MaterialName: `Металический сайдинг`,
		ID:           "metalPrice",
		PictureURL:   "/assets/img/promo/metall.jpg",
		UnitName:     "кв.м.",
		YoutubeID:    "p_G9w0iF6yU",
	}

	if err := tmpl.Execute(w, data); err != nil {
		loggi.Fatal(err)
	}
}
